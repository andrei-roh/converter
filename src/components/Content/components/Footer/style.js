import styled from 'styled-components';

export const FooterBlock = styled.div(({ theme }) => ({
  display: 'none',

  '@media (min-width: 769px)': {
    padding: '26px 0',
    width: '100%',
    color: theme.header,
    backgroundColor: theme.main,
    display: 'flex',
    justifyContent: 'center',
    alignItems: 'center',
    marginTop: '15px',
    position: 'fixed',
    bottom: 0,
    zIndex: 2,
  },
}));
