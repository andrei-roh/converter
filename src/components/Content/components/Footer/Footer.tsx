import { FooterBlock } from './style';
import Link from '@material-ui/core/Link';

const Footer = () => {
  const getCurrentYear = () => {
    const today = new Date();
    return today.getFullYear();
  };
  return (
    <FooterBlock>
      <Link
        underline="none"
        color="inherit"
        href="https://github.com/andrei-roh"
        target="_blank"
      >
        {getCurrentYear()}
      </Link>
    </FooterBlock>
  );
};

export default Footer;
