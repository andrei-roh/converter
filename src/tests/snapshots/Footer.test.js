import { shallow } from 'enzyme';
import Footer from '../../components/Content/components/Footer/Footer';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<Footer />);
});

describe('<Footer /> rendering', () => {
  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
