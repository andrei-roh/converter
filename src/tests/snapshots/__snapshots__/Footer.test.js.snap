// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> rendering renders correctly 1`] = `ShallowWrapper {}`;
